usaspending_api/api_contracts/contracts/v2/search/spending_by_category/federal_account.md

```diff
@@ -0,0 +1,193 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Spending By Federal Account [/api/v2/search/spending_by_category/federal_account/]
+
+This endpoint supports the advanced search page and allow for complex filtering for specific subsets of spending data.
+
+## POST
+
+This endpoint returns a list of the top results of Federal Accounts sorted by the total amounts in descending order.
+
++ Request (application/json)
+    + Attributes (object)
+        + `filters` (required, FilterObject)
+            The filters to find with said category
+        + `limit`: 5 (optional, number)
+            The number of results to include per page
```

Comment:
If this defaults to 5 if not provided, can we include `+ Default: 5` here? (Similar for page number)